pennylane/transforms/__init__.py

```diff
@@ -193,7 +193,7 @@
 from .compile import compile
 from .decompositions import zyz_decomposition, two_qubit_decomposition
 from .defer_measurements import defer_measurements
-from .hamiltonian_expand import hamiltonian_expand
+from .hamiltonian_expand import hamiltonian_expand, sum_expand
```

Comment:
I just noticed that `sum_expand` is not displayed in the docs since the transforms are manually created. I would add it just below `hamiltonian_expand` in line 150

Reply:
Done!